blueprints/module-unification-addon/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+const addonBlueprint = require('../addon');
+
+module.exports = Object.assign({}, addonBlueprint, {
```

Comment:
This is where it all happens. The only difference with the generate of standard addons is the description and the files. I couldn't find a better way of "inheriting" from an existing blueprint.